src/lib/components/intl-tel-input.component.spec.ts

```diff
@@ -7,7 +7,7 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
good catch ! The linter should have warned...

Reply:
Maybe updating all packages will help